### DISCUSSION TITLE
Deploy bot with webhooks

### DISCUSSION THREAD

**Discussion by ozzyst** [aiogram/aiogram#505] [Q&A] [answered]
Hello everyone

Could you help me with deploying the aiogram with a webhook? What is the best way? I read a lot of comments about it, but I didn't find answer for my question.

1. Should I use *corn between nginx and aiogram? Why?
2. How could I configure the *corn, if I didn't have app in my main file? I mean I start the bot via start_webhook(...)
3. Any example?

**Comment by ozzyst** [comment]
Any chance to help me?

**Comment by Olegt0rr** [comment, marked as answer]
1. As you wish. You can use unlimited amount of proxying webservers. IMHO nginx is enough for most cases.
2. If you wanna use app, look at [this example](https://github.com/aiogram/aiogram/blob/dev-2.x/examples/separate_api_route_example.py)
3. [Basic webhook example](https://github.com/aiogram/aiogram/blob/dev-2.x/examples/webhook_example.py)
